README.md

```diff
@@ -1 +1,184 @@
+
+```
+sudo apt-get install ruby-full
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+exec $SHELL
+
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bashrc
+exec $SHELL
+
+rbenv install 3.0.1
+rbenv global 3.0.1
+ruby -v
+```
+
+### Setup
+
+To get a local copy up and running, follow these steps.
+Choose the directory on your local machine where you want to copy project. For example:
+
+```
+cd /home/user/name-of-your-directory
+```
+
+Clone the project using one of the options.
+
+Using SSH-key:
+
+```
+git clone git@github.com:Alejuda/morse-decode.git
+```
+
+Using HTTPS:
+
+```
+git clone https://github.com/Alejuda/morse-decode.git
+```
+
+### Usage
+
+1. Ensure that Ruby is installed on your system.
+
+2. Open a terminal or command prompt.
+
+3. Run the following command to run the file:
+
+```
+ruby morse-decode.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Authors <a name="authors"></a>
+
+👤 **Romans S.**
+
+- GitHub: [@romans-adi](https://github.com/romans-adi/)
+- Twitter: [@obj583](https://twitter.com/obj583/)
+- LinkedIn: [Romans Špiļaks](https://www.linkedin.com/in/obj513/)
+
+👤 **Ignacio Fino**
+
+- GitHub: [@alejuda](https://github.com/Alejuda)
+- Twitter: [@nacho_fino](https://twitter.com/nacho_fino)
+- LinkedIn: [Ignacio Fino](https://www.linkedin.com/in/ignacio-fino-320916209)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## Future Features <a name="future-features"></a>
+
+- **Expand the capabilities of the Morse Decoder by adding a feature that allows users to input English text and encode it into Morse code.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## Contributing <a name="contributing"></a>
+
+We welcome contributions to enhance the functionality and user experience of the Morse Deecoder project. If you have any ideas, suggestions, or bug reports, feel free to open an issue or submit a pull request. Let's help the animals together!
+
+If you'd like to contribute to this project, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Make your changes and commit them with descriptive commit messages.
+4. Push your changes to your forked repository.
+5. Submit a pull request to the main repository, explaining your changes in detail.
+
+Please adhere to the coding conventions and guidelines specified in the project.
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## Support <a name="support"></a>
+
+If you encounter any issues or have any questions or suggestions, please open an issue on the [issue tracker](../../issues/).
```

Comment:
![image](https://github.com/Alejuda/morse-decode/assets/78251927/893a8694-0650-4df4-be92-575e4f343e98)

Please fix the broken link to the issue page